source/index.txt

```diff
@@ -20,8 +16,76 @@ MongoDB in Ruby. To work with {+odm+} from the command line using
    Secure Your Data </security>
    Integrations & Tools </integrations-tools>
    API Documentation </api>
-   What's New </whats-new>
    Compatibility </compatibility>
+   What's New </whats-new>
    Issues & Help </issues-and-help>
-   /additional-resources
-   /ecosystem
+   View the Source <https://github.com/mongodb/mongoid>
+
+Introduction
+------------
+
+Welcome to the documentation site for {+odm+}. {+odm+} is the officially
+supported object-document mapper (ODM) for MongoDB in {+language+}.
+
+You can add {+odm+} to your application to connect your {+language+} to
+MongoDB. Install it by adding it to your project's ``Gemfile`` file or
+set up a runnable project by following one of the Quick Start guides.
+
+Quick Start
+-----------
+
+Learn how to establish a connection to MongoDB Atlas and begin
+working with data by following one of the following guides:
+
+- :ref:`mongoid-quick-start-rails`
+- :ref:`mongoid-quick-start-sinatra`
+
+Configuration
+-------------
+
+To learn how to configure different options in your {+odm+} application,
+see the :ref:`mongoid-configuration` section.
+
+Interact with Data
+------------------
+
+To learn how to use {+odm+} to interact with your MongoDB data,
+see the :ref:`mongoid-interact-data` section.
+
+Model Your Data
+---------------
+
+To learn how to model your MongoDB data as {+odm+} models,
+see the :ref:`mongoid-data-modeling` section.
+
+Secure Your Data
+----------------
+
+To learn how to secure your data by using encryption,
+see the :ref:`mongoid-security` section.
+
+Integrations & Tools
+--------------------
+
+To learn how to add {+odm+} to an existing application,
+see the :ref:`mongoid-integrations-tools` section. This section also
+includes information about the Rails framework and other resources.
+
+Compatibility
+-------------
+
+To learn about the versions of the MongoDB Server, {+language+}
+language, {+ruby-driver+}, and Rails that are compatible with each
```

Comment:
s: wording (not sure if there's a mdb-server constant)

```suggestion
To learn about the versions of {+mdb-server+}, {+language+},
the {+ruby-driver+}, and Rails that are compatible with each
```

Reply:
I can add the constant